.storybook/README.md

```diff
@@ -0,0 +1,5 @@
+# Storybook Configuration
+
+This folder contains configuration items for Storybook.
+
+The /assets folder contains symlinks to the root asset folders so that the `yarn build-storybook` command will correctly obtain and use the assets from their present location (also used by the `yarn storybook` command to serve static assets).
```

Comment:
Did you forgot to commit the `assets` folder?

Reply:
so I actually removed it because it was huge and looked very webapp-specifif?